src/witan/send/driver/transition.clj

```diff
@@ -0,0 +1,76 @@
+(ns witan.send.driver.transition
```

Comment:
Not too concerned, but I feel like this should be `transitions`

Reply:
done